connectivity/FEATURE_BLE/source/cordio/source/PalSecurityManagerImpl.cpp

```diff
@@ -444,6 +482,7 @@ ble_error_t PalSecurityManager::cancel_pairing(
 ble_error_t PalSecurityManager::get_random_data(byte_array_t<8> &random_data)
 {
     SecRand(random_data.data(), random_data.size());
+    tr_info("Stack generated random data: %s", to_string(random_data));
```

Comment:
don't think this is useful, if the data is relevant the caller will log it

Reply:
That still useful, the caller will log things like `set CSRK: .....` It doesn't say much about the origin of the bytes. 
I'd keep it, it help me in identifying the issue fixed by https://github.com/ARMmbed/mbed-os/pull/14125